test/TestInvestorLedger.sol

```diff
@@ -0,0 +1,78 @@
+pragma solidity ^0.4.11;
+
+import "truffle/Assert.sol";
+import "truffle/DeployedAddresses.sol";
+import "../contracts/loans/InvestorLedger.sol";
+import "../contracts/tokens/PrintableToken.sol";
+
+
+contract TestInvestorLedger {
+    uint256 totalLoanNeeded = 100;
+    uint16 interestPermil = 200;
+    uint256 EMPTY = 0;
+    uint256 PRINT_VALUE = 100;
+    PrintableToken collateralToken = createMockToken("collateralToken");
+    PrintableToken loanToken = createMockToken("loanToken");
+    TestPerson investor = new TestPerson(loanToken, collateralToken);
+    TestPerson borrower = new TestPerson(loanToken, collateralToken);
+    InvestorLedger.Ledger testLedger = InvestorLedger.openAccount(
+        collateralToken, loanToken, address(borrower), totalLoanNeeded, interestPermil);
+
+    function createMockToken(string memory tokenName)  returns (PrintableToken) {
```

Comment:
Is this really necessary? Seems like a bit too DRY, you're only calling this twice and this only really binds two default arguments.

Reply:
Doesn't it look nice :) ?
```
PrintableToken collateralToken = createMockToken("collateralToken");
PrintableToken loanToken = createMockToken("loanToken");
```

Reply:
Not worth the extra code complexity.

Reply:
Let's leave it as it is

Reply:
or maybe not, u r right, the simpler the better. Removed createMockToken function